include/seqan3/range/views/minimiser.hpp

```diff
@@ -440,25 +578,29 @@ namespace seqan3::views
  * \{
  */
 
-/*!\brief                           Computes minimisers for a range of comparable values. A minimiser is the smallest
- *                                  value in a window.
- * \tparam urng_t                   The type of the range being processed. See below for requirements. [template
- *                                  parameter is omitted in pipe notation]
- * \param[in] urange                The range being processed. [parameter is omitted in pipe notation]
- * \param[in] window_values_size    The number of values in one window.
- * \returns                         A range of std::totally_ordered where each value is the minimal value for one window.
- *                                  See below for the properties of the returned range.
+/*!\brief Computes minimisers for a range of comparable values. A minimiser is the smallest value in a window.
+ * \tparam urng1_t The type of the first range being processed. See below for requirements. [template
+ *                 parameter is omitted in pipe notation]
+ * \tparam urng2_t The type of the range second being processed. See below for requirements. [template
```

Comment:
```suggestion
 * \tparam urng2_t The type of the second range being processed. See below for requirements. [template
```

Reply:
But there is no description of the requirements?
I guess, we should add a second column which denotes that the second range is optional.